frame/election-provider-support/src/lib.rs

```diff
@@ -391,7 +391,7 @@ pub trait ElectionProviderBase {
 				if desired_targets <= Self::MaxWinners::get() {
```

Comment:
Try rewriting this function with `and_then` please.

Reply:
Thanks for the nudge.